pkgs/applications/version-management/jiri/default.nix

```diff
@@ -0,0 +1,81 @@
+{ stdenv
+, lib
+, fetchurl
+, fetchzip
+, unzip
+, autoPatchelfHook
+, version ? "unstable-2021-06-15"
```

Comment:
```suggestion
```
unused and most likely does not work like you think it does.

Reply:
Done